src/pages/Farm/StakeDialog.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Box,
+  Button,
+  DialogContent,
+  Stack,
+  Tab,
+  Tabs,
+  Typography,
+} from "@mui/material"
+import React, { useState } from "react"
+import Dialog from "../../components/Dialog"
+import TokenInput from "../../components/TokenInput"
+
+interface StakeDialogProps {
+  open: boolean
+  onClose: () => void
+  farmName: string
+}
+
+export default function StakeDialog({
+  open,
+  farmName,
+  onClose,
+}: StakeDialogProps): JSX.Element {
+  const [stakeStatus, setStakeStatus] = useState<"stake" | "unStake">("stake")
+  return (
+    <Dialog open={open} onClose={onClose} fullWidth maxWidth="xs">
+      <DialogContent sx={{ mt: 3 }}>
+        <Stack spacing={3}>
+          <Typography variant="h1" textAlign="center">
+            {farmName}
+          </Typography>
+          <Typography>
+            Stake your LP token and collect SDL incentives.
+          </Typography>
+          <Stack
+            direction="row"
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <Box>
+              <Typography>LP Staked</Typography>
+              12334578.12
+            </Box>
+            <Box>
+              <Typography>Rewards</Typography>
+              12334578.12
+            </Box>
+            <Button variant="outlined" size="large">
+              Claim
+            </Button>
+          </Stack>
+          <Tabs
+            value={stakeStatus}
+            variant="fullWidth"
+            onChange={(_, newValue) => setStakeStatus(newValue)}
+          >
+            <Tab value="stake" label="Stake" />
+            <Tab value="unStake" label="Unstake" />
```

Comment:
`value=unstake`?

Reply:
This is how `Tabs` works in MUI. It is a little tricky, but when click `Tab` the newValue on the function of `onChange` in `Tabs` will transfer this value

Reply:
that's fine, I just wanted to make sure you saw the casing on `unStake -> unstake`